root/types/releasegroup.js

```diff
@@ -20,6 +20,7 @@ declare type ReleaseGroupT = $ReadOnly<{
   ...TypeRoleT<ReleaseGroupTypeT>,
   +cover_art?: ArtworkT,
   +firstReleaseDate: string | null,
+  +hasCoverArt?: boolean,
```

Comment:
This is always outputted, so you can remove the `?`:
```suggestion
  +hasCoverArt: boolean,
```